signatures.md

```diff
@@ -10,40 +10,37 @@ We use signatures in filecoin to verify *something* was done by *someone*. For e
 - Tickets (Signature of proof - [Mining](mining.md)
 - Block signature (Signature over all data in the block - done by block leader)
 
-# What signatures affect
+## Filecoin components that signatures affect
 
-What uses them
+- [Messages](https://github.com/filecoin-project/specs/blob/master/data-structures.md)
+- [Block validation](https://github.com/filecoin-project/specs/blob/master/mining.md)
+- [Tickets](https://github.com/filecoin-project/specs/blob/master/mining.md)
 
-- Messages [TODO: Link to messages spec]
-- Block validation
-- Tickets which inform leader election in Expected Consensus (EC) [TODO: link to EC spec]
+Note that messages between actors are not signed. This is an unfortunate namespace collision. Messages between actors are always spawned by a message from a user. In this way, signed messages are containers for messages between actors.
 
-Note that messages between actors are not signed this is because messages between actors are always spawned by a message from a user -- which are singed by that user. 
+## Dependencies
 
-Filecoin specific reliance
+- Elliptic curve implementations
+  - ECDSA with secp256k1 (BitCoin-style elliptic curve)
+  - BLS12-381 for BLS signatures and aggregation. 
+    - Described as *Variant 1* in [IETF BLS Signature Scheme](https://tools.ietf.org/html/draft-boneh-bls-signature-00)
+    - Small signature size
+    - Larger public keys
+- Signature size limits
+  - ECDSA - 64 bytes + 1 byte for public key recovery
+  - BLS Signatures - 48 Bytes
+- Avg. processing power available to a CPU 
+  - Affects ability to verify signature aggregates
+  - Each signature or collection of signatures should be validated with no impact to the consumption of the chain
+- `Gas` cost - `Gas(ECDSA) > Gas(BLS)`
```

Comment:
please clarify which operations are more costly than others, eg `Gas(ECDSA_Sign)`

Reply:
I don't have this information. Ping @whyrusleeping : can you answer this?